inc/middleware/nx_verified_telemetry.h

```diff
@@ -75,24 +86,28 @@ typedef struct NX_VERIFIED_TELEMETRY_DB_TAG
  * @param[in] enableVerifiedTelemetry User specified value to set Verified Telemetry active or inactive, can also be configured
  * during runtime from a writable Digital Twin property.
  * @param[in] device_driver The platform specific device driver components for interacting with the device hardware.
+ * @param[in] scratch_buffer Pointer to buffer passed from application layer, used for fingerprint calculation/storage.
  *
  * @retval NX_AZURE_IOT_SUCCESS upon success or an error code upon failure.
  */
 UINT nx_vt_init(NX_VERIFIED_TELEMETRY_DB* verified_telemetry_DB,
     UCHAR* component_name_ptr,
     bool enable_verified_telemetry,
-    VT_DEVICE_DRIVER* device_driver);
+    VT_DEVICE_DRIVER* device_driver,
+    CHAR* scratch_buffer);
```

Comment:
>CHAR* scratch_buffer); [](http://example.com/codeflow?start=4&length=22)

coming back to the recommended buffer size comment above, if people pass in a buffer different from the size we request, then we should also ask them to pass in the buffer size so run time sanity checks can happen.

Reply:
fixed